src/App.tsx

```diff
@@ -69,7 +69,7 @@ const App = (_props: Props) => {
       setMetricsIndex,
       setGovernedParamsIndex,
       setInstanceIds,
-    });
+    }).catch((err: Error) => console.error('got watchContract err', err));
```

Comment:
Worth alerting here too? (Maybe if/when we get time we can add a toast instead of alert)

Reply:
I figured if this part fails there's not anything they can do. for the signing error they can get more funds.